src/NuGet/Microsoft.CodeAnalysis.CSharp.CodeStyle.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="3.3">
+    <id>Microsoft.CodeAnalysis.CSharp.CodeStyle</id>
+    <description>
+      .NET Compiler Platform ("Roslyn") code style analyzers for C#.
+    </description>
+    <dependencies>
+    </dependencies>
+
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <version>$version$</version>
+    <authors>$authors$</authors>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>$tags$</tags>
+    <serviceable>true</serviceable>
+    <developmentDependency>true</developmentDependency>
+  </metadata>
+  <files>
+    <file src="Dlls\CSharpCodeStyle\Microsoft.CodeAnalysis.CSharp.CodeStyle.dll" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CSharpCodeStyle\Microsoft.CodeAnalysis.CSharp.CodeStyle.xml" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CSharpCodeStyleFixes\Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes.dll" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CSharpCodeStyleFixes\Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes.xml" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CodeStyle\Microsoft.CodeAnalysis.CodeStyle.dll" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CodeStyle\Microsoft.CodeAnalysis.CodeStyle.xml" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CodeStyleFixes\Microsoft.CodeAnalysis.CodeStyle.Fixes.dll" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CodeStyleFixes\Microsoft.CodeAnalysis.CodeStyle.Fixes.xml" target="analyzers\dotnet\cs" />
+    <file src="$thirdPartyNoticesPath$" target="" />
+
+    <!-- Scripts -->
+    <file src="..\..\src\Setup\PowerShell\install.ps1" target="tools\" />
```

Comment:
Yes you need these if someone's installing the package to a packages.config-based project which is still the majority of projects out there.